SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -4,7 +4,8 @@ import Foundation
 
 public final class SwiftGenStoryboardEnumBuilder {
     private typealias Scene = (storyboardID: String, customClass: String?)
```

Comment:
We should probably rename that tuple to something else for consistency, to give it a name that can both fit for Scenes and Segues (and then use `identifier` for the first entry name, instead of `storyboardID`)